site/profile/manifests/puppet/master/se_gitbook.pp

```diff
@@ -1,7 +1,44 @@
 class profile::puppet::master::se_gitbook (
-  $gitbook_port = '4000',
+  $gitbook_port    = '4000',
+  $gitbook_wwwroot = '/var/www/se_gitbook',
+  $gitbook_source  = 'https://s3-us-west-2.amazonaws.com/tse-builds/gitbooks/',
+  $gitbook_release = 'sebook.tar',
```

Comment:
Wondering if we set this to a `pick_default($::gitbook_file,'sebook.tar')` (or something similar) so that in the future we can just drop a fact during the master build to control which version of the docs are deployed.  Thoughts? @murdok5